underfs/tos/src/main/java/alluxio/underfs/tos/TOSUnderFileSystem.java

```diff
@@ -348,6 +389,30 @@ protected String getRootKey() {
     return Constants.HEADER_TOS + mBucketName;
   }
 
+  /**
+   * Creates an TOS {@code ClientConfiguration} using an Alluxio Configuration.
+   * @param alluxioConf the TOS Configuration
+   * @return the TOS {@link TransportConfig}
+   */
+  public static TransportConfig initializeTOSClientConfig(
+      AlluxioConfiguration alluxioConf) {
+    int readTimeoutMills = (int) alluxioConf.getMs(PropertyKey.UNDERFS_TOS_READ_TIMEOUT);
```

Comment:
if you want to use `alluxioConf.getMs`, use the durationBuilder instead of intBuilder.
if you use the intBuilder, please use `getInt`.